src/helm/proxy/models.py

```diff
@@ -112,11 +112,10 @@ def engine(self) -> str:
 # Over time, we should add more information there.
 
 ALL_MODELS = [
-    # Local Model
     Model(
-        group="neurips",
-        name="neurips/local",
-        tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, GPT2_TOKENIZER_TAG],
+        group="sagemaker",
```

Comment:
FYI, after #1861 is merged (probably by tomorrow), you should be able to do something like:

`prod_env/model_deployments.yaml`:

```
model_deployments:
  - name: my-org/my-model-1
    window_service_spec:
      class_name: "helm.benchmark.window_services.gpt2_window_service.GPT2WindowService"
      args: {}
    client_spec:
      class_name: "helm.proxy.clients.sagemaker_client.SageMakerClient"
      args: {
        sagemaker_endpoint_name: endpoint-1
      }
  - name: my-org/my-model-2
    window_service_spec:
      class_name: "helm.benchmark.window_services.gpt2_window_service.GPT2WindowService"
      args: {}
    client_spec:
      class_name: "helm.proxy.clients.sagemaker_client.SageMakerClient"
      args: {
        sagemaker_endpoint_name: endpoint-2
      }
```

This does break reverse compatibility with the old config format.